lab-server/test/helpers/keycloak-mock.ts

```diff
@@ -5,23 +5,42 @@ interface KeycloakMockHelper {
     mockToken: string;
 }
 
-export const getKeycloakMockHelper = async (): Promise<KeycloakMockHelper> => {
-    const keycloakMock = await KeycloakMock.createMockInstance({
+/*
+If the argument isUserValid is set to false, then the function returns a KeycloakMockHelper with
+an invalid token, otherwise returns an an instance with a valid token
+*/
```

Comment:
For multi-line comments it is more common to use "block-style" comments in JS. See [this](https://blog.shhdharmen.me/comments-usage-and-best-practices-in-javascript#multi-line-or-block) for reference